src/HospitalLibrary/Appointments/AppointmentService.cs

```diff
@@ -62,29 +69,36 @@ public async Task<IEnumerable<TimeIntervalWithDoctorDTO>> GetTimeIntervalsForRec
                 await GetTimeIntervalsWithDoctorForDate(doctor, dateIterator, possibleTimeIntervals);
                 dateIterator = dateIterator.AddDays(1);
             }
+
             return possibleTimeIntervals;
         }
-        public async Task<IEnumerable<TimeIntervalWithDoctorDTO>> GetTimeIntervalsForRecommendationDatePriority(int patientId, string speciality, DateTime start,DateTime end)
+
+        public async Task<IEnumerable<TimeIntervalWithDoctorDTO>> GetTimeIntervalsForRecommendationDatePriority(
+            int patientId, string speciality, DateTime start, DateTime end)
```

Comment:
Same thing here, except that you can wrap all of these parameters in one dto. I think TimeInterval is enough here